app/models/ext_management_system.rb

```diff
@@ -414,6 +416,14 @@ def self.ems_infra_discovery_types
     @ems_infra_discovery_types ||= %w(virtualcenter scvmm rhevm)
   end
 
+  def disable_ems
```

Comment:
I think these should be bang methods if they update the object in place (see [toggle](http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-toggle) for AR methods.

Also maybe strip `_ems` ? 

```ruby
ems.enable!
ems.disable!
```